### ISSUE TITLE
Pagination for MyClasses

### ISSUE THREAD

**Issue by nicksavov** [OSTraining/OSCampus#144]
https://www.ostraining.com/myclasses/ doesn't appear to have pagination. Mine goes to the G classes and ends there with Google Analytics as the last one.

**Comment by billtomczak** [comment]
Yes - is possible but not implemented. For now, like guru you should see all classes.

**Comment by nicksavov** [comment]
No problem. Most people won't registered in as many classes as I am
